scripts/src/chartprreview/chartprreview.py

```diff
@@ -14,27 +14,34 @@
 except ImportError:
     from yaml import Loader, Dumper
 
+def write_error_log(directory, *msg):
+    with open(os.path.join(directory, "errors"), "w") as fd:
+        for line in msg:
+            print(line)
+            fd.write(line)
+            fd.write("\n")
 
-def get_modified_charts(api_url):
+def get_modified_charts(directory, api_url):
     files_api_url = f'{api_url}/files'
     headers = {'Accept': 'application/vnd.github.v3+json'}
     r = requests.get(files_api_url, headers=headers)
     pattern = re.compile(r"charts/(\w+)/([\w-]+)/([\w-]+)/([\w\.]+)/.*")
-    count = 0
     for f in r.json():
         m = pattern.match(f["filename"])
         if m:
             category, organization, chart, version = m.groups()
             return category, organization, chart, version
 
-    return "", "", "", ""
+    msg = "[ERROR] The files modified are not part of any chart"
+    write_error_log(directory, msg)
+    sys.exit(1)
 
-
-def verify_user(username, category, organization, chart):
+def verify_user(directory, username, category, organization, chart):
     data = open(os.path.join("charts", category, organization, chart, "OWNERS")).read()
     out = yaml.load(data, Loader=Loader)
     if username not in [x['githubUsername'] for x in out['users']]:
-        print("User doesn't exist in list of owners:", username)
+        msg = f"[ERROR] User doesn't exist in list of owners: {username}"
```

Comment:
Maybe "${username} who submitted the pull request is not included in the list of owners"

Reply:
Perhaps "{usernmae} is not allowed to submit the chart on behalf of {partner}"

Reply:
I have taken @pedjak 's suggestion here: 229bd4e5c3b304f1c759c7b5b8f51971defae155